internal/pkg/cli/env_deploy.go

```diff
@@ -123,6 +136,16 @@ func (o *deployEnvOpts) Execute() error {
 	if err != nil {
 		return err
 	}
+	// TODO: Put into a new DynamicEnvironmentManifest struct in "manifest" package
```

Comment:
is this getting resolved in your next PR? (if not and it's just like a "future would-be-nice TODO, let's get rid of it)

Reply:
@iamhopaul123 can tell you more about it, I don't know what this means to do, and won't be doing myself